kotlin-lotto/src/main/kotlin/lotto/_enums/LottoResult.kt

```diff
@@ -0,0 +1,34 @@
+package lotto._enums
+
+import lotto.util.decimalFormat
+
+enum class LottoResult(
+    private val num: Int,
+    private val isBonusCorrect: Boolean?,
+    val price: Int,
+) {
+    THREE_CORRECT(3, null, 5000),
+    FOUR_CORRECT(4, null, 50000),
+    FIVE_CORRECT(5, false, 1500000),
```

Comment:
```suggestion
    FIVE_CORRECT(5, false, 1_500_000),
```

위처럼 하면 가독성이 조금 더 높아질 것 같습니다~